src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -746,10 +746,9 @@ public void windowsContainer() throws Exception {
         r.assertLogContains("C:\\Users\\ContainerAdministrator", b);
         r.assertLogContains("got stuff: some value", b);
     }
-
-    @Ignore(
-            "TODO aborts, but with “kill finished with exit code 9009” and “After 20s process did not stop” and no graceful shutdown")
+    
     @Test
+    @Ignore("Can not be fixed in kubernetes, see https://github.com/jenkinsci/kubernetes-plugin/pull/1724#discussion_r2287512410")
```

Comment:
OK but is there some problem this PR _is_ fixing which can be demonstrated in an automated test? (Make a note of the failure log with `src/main/java/` reverted.)